Environment_Engine/Create/Gains.cs

```diff
@@ -117,13 +117,30 @@ public static People People(double sensible = 0.0, double latent = 0.0, Profile
                 Name = name,
             };
         }
-
+        [Description("Create a Plug Gain object which can be attributed to a space")]
+        [Input("sensible", "The sensible gain load for the plug gain, default 0.0")]
+        [Input("profile", "The profile for this gain being active, default null")]
+        [Input("radiantFraction", "The fraction of radiance from this people gain, default 0.0")]
+        [Input("viewCoefficient", "The view coefficient of this plug gain, default 0.0")]
+        [Input("name", "The name of this plug gain, default empty string")]
+        [Output("plugGain", "The plug gain object to associate to a space")]
+        public static Plug Plug(double sensible = 0.0, Profile profile = null, double radiantFraction = 0.0, double viewCoefficient = 0.0, string name = "")
+        {
+            return new Plug
+            {
+                Sensible = sensible,
+                Profile = profile,
+                RadiantFraction = radiantFraction,
+                ViewCoefficient = viewCoefficient,
+                Name = name,
+            };
+        }
```

Comment:
Please add a blank line here after this bracket to separate this method from the next one